examples/experimental/ProductionMonitoring/StreamTable.md

```diff
@@ -0,0 +1,88 @@
+# Weave StreamTable
+
+Log and explore some basic StreamTables now in the [interactive notebook version](../ProductionMontoring/stream_table_api.ipynb) of these docs.
+
+A Weave StreamTable object enables continuous streaming of data from an application or service to W&B. You can append data repeatedly to the same StreamTable object with `.log([your data rows])` and build dynamic visualizations from the streaming data, like example to recognize MNIST digits hand-drawn in an [interactive Jupyter notebook](../ProductionMonitoring/ProductionMonitoringConceptualOverview.ipynb).
+
+![small_prodmon_board](../../../docs/assets/mnist_pm_draw_hover.png)
+ 
+## Create a StreamTable
+
+The only required argument to create a StreamTable is the name of the StreamTable object. 
+
+```python
+from weave.monitoring import StreamTable
+st = StreamTable("my_entity_name/my_project_name/my_table_name")
+```
+If an entity (W&B username or shared team name) is not provided, this will attempt to default to the current logged-in entity.
+
+## Log data to a StreamTable
+
+Call `.log()` to add rows to a StreamTable:
+
+```python
+st.log({"one_column_name" : "value_a", "another_column_name" : 7})
+st.log([
+    {"one_column_name" : "value_b", "another_column_name" : 19},
+    {"one_column_name" : "value_c", "another_column_name" : 28},
+    {"one_column_name" : "value_d", "another_column_name" : 36}])
+```
+`.log()` accepts a single dictionary or a list of dictionaries, where each dictionary entry corresponds to one row of the table. In each dictionary, the keys are column names and the values are the corresponding cell values.
+
+## Visualize the StreamTable
+
+The first call to `.log()` will return a Weave Panel URL, where you can view, edit, and save the resulting StreamTable as a Weave Board, of the form:
+
+View data at : https://weave.wandb.ai/?exp=get%28%0A++++%22wandb-artifact%3A%2F%2F%2Fstacey%2Fmesa%2Fmy_stream_table%3Alatest%2Fobj%22%29%0A++.rows
+
+![prodmon_tiny_table](../../../docs/assets/small_stream_table.png)
+
+
+Subsequent log calls will silently append these rows to the StreamTable instance.
+
+In a notebook, the StreamTable variable on a line by itself will return a Weave Panel view of the StreamTable. The StreamTable will contain all the logged columns and their values, as well as a `timestamp` column indicating when the row was logged. By default, rows will be ordered by oldest first. You can modify a StreamTable Panel from the UI to sort by columns, group by column values, filter for specific ranges or values, etc.
+
+**Note:** If you would like to customize and save a specific view of a StreamTable Panel, open the StreamTable Panel in a new window as a Board and edit/save a Board from this seed panel. There are two options to achieve this:
+* via the weave.wandb.ai/?exp=... URL
```

Comment:
i don't think this is really useful cuz users can't really type a nice URL here. I'd just remove this and make gifs showing how to do it from the table view (which they know how to get to from the link and section above) or your screen shot below (maybe change it to a gif) for a notebook.

Reply:
thank you, this gif/flow will be top priority in next docs—I hope this is an ok start for now